components/backdrop/backdrop.js

```diff
@@ -8,13 +8,27 @@ const reduceMotion = matchMedia('(prefers-reduced-motion: reduce)').matches;
 const scrollKeys = [];
 let scrollOverflow = null;
 
+/**
+ * The `d2l-backdrop` element is a web component to display a semi-transparent backdrop behind a specified sibling element. It also hides elements other than the target from assistive technologies by applying `role="presentation"` and `aria-hidden="true"`.
+ */
 class Backdrop extends LitElement {
 
 	static get properties() {
```

Comment:
This file uses an attribute `slow-transition` in the css (https://github.com/BrightspaceUI/core/blob/e5a10aaaeaeb37ca4c07038481e281b2a39e5c65/components/backdrop/backdrop.js#L49) that's called out in the readme but isn't listed in properties so it doesn't show up in the custom-elements.json. Should it get added to properties?